rlbot/interface.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import time
```

Comment:
unused import

Reply:
`time.time()` is used in `connect`. Will fix to make it consistent with `time.sleep()` usage.